controllers/database.go

```diff
@@ -27,13 +27,14 @@ import (
 
 	"errors"
 
+	"os"
```

Comment:
I try to leave stuff like this out of the commit, since it's not part of the change. Usually I do a final pass with "meld" before pushing and undo any stuff like this that my IDE did.